osu.Game/Rulesets/Mods/ModEasy.cs

```diff
@@ -26,5 +28,25 @@ public void ApplyToDifficulty(BeatmapDifficulty difficulty)
             difficulty.DrainRate *= ratio;
             difficulty.OverallDifficulty *= ratio;
         }
+
+        public void ApplyToScoreProcessor(ScoreProcessor scoreProcessor)
+        {
+            Lives = 2;
+            scoreProcessor.Health.ValueChanged += valueChanged =>
+            {
+                if (scoreProcessor.Health.Value == 0)
+                {
+                    if (Lives != 0)
+                    {
+                        Lives--;
+                        scoreProcessor.Health.Value = 100;
+                    }
+                    else
+                    {
+                        Lives = 2;
```

Comment:
What's the reason for having this here?

Reply:
Thats actually a bit of disposable, old code. tested here now and it can be removed without affecting the end result.